webapi/src/main/scala/org/knora/webapi/responders/ResponderManager.scala

```diff
@@ -185,60 +186,70 @@ class ResponderManager(appActor: ActorRef) extends Actor with ActorLogging {
      */
     protected final def makeDefaultOntologiesResponderV2: OntologyResponderV2 = new OntologyResponderV2(responderData)
 
+    /**
+     * Subclasses of the can override this member to substitute a with a custom implementation instead of the default ontologies responder.
+     */
+    protected val ontologiesResponderV2: OntologyResponderV2 = makeDefaultOntologiesResponderV2
+
     /**
      * Constructs the default [[SearchResponderV2]].
      */
     protected final def makeDefaultSearchResponderV2: SearchResponderV2 = new SearchResponderV2(responderData)
 
+    /**
+     * Subclasses of the can override this member to substitute a with a custom implementation instead of the default search responder.
+     */
+    protected val searchResponderV2: SearchResponderV2 = makeDefaultSearchResponderV2
+
     /**
      * Constructs the default [[ResourcesResponderV2]].
      */
     protected final def makeDefaultResourcesResponderV2: ResourcesResponderV2 = new ResourcesResponderV2(responderData)
 
     /**
-     * Constructs the default [[ValuesResponderV2]].
+     * Subclasses of the can override this member to substitute a with a custom implementation instead of the default resources responder.
```

Comment:
What does that mean?

Reply:
The same typo was in all the existing comments in `ResponderManager`, we just copied and pasted it. :) Fixed now.